### DISCUSSION TITLE
Other profiles for P1P and 0.6mm Nozzle?

### DISCUSSION THREAD

**Discussion by gasprints** [OrcaSlicer/OrcaSlicer#854] [Q&A]
I was wondering if anyone has successfully made more profiles for the P1P using a 0.6mm nozzle?  It seems theres only a 0.3mm layer hight profile and I was really hoping for larger layer heights.

**Comment by julie777** [comment]
I have a suggestion regarding profiles, of all kinds. 
Let's create a separate repo for profiles independent from the slicer with the appropriate directory structure. 

Adding a new printer might need slicer code changes, but the other information in profiles doesn't. There is not reason to couple them to the slicer, except in the sense that they use settings that the slicer recognizes. 

There are many benefits of separating the profiles and gcode for that matter:
  - profiles can be added or modified without a slicer change, and have a much simpler review before committing to the repo.
  - profiles would be easy to download at any time
  - profiles can be organized in a way that makes it easy for a user to find and download just what they want
  - profiles do not need a release process.
 
The best part would be that users could contribute all there best settings and they would be stored in one place for others to use. (I am sure that by now someone has a profile for the P1P and a .6mm nozzle. They just don't have a good way to share it.)

Something like this?

OrcaProfiles
  - printer
      - BambuLab
          - P1P 
              - .4mm nozzle
              - .6mm nozzle
              - start_gcode
              - fast_start_gcode
              - etc.
          - X1
      - other brands and printers 
  - filament
      - pla
          - Hatchbox
          - etc.
      - abs
      - etc.   
  - print
      - .2mm speed


I am not sure what the best organization would be, but it would be nice to try a new filament and just look to find a profile that works. Or, look for a printing profile that works with a .6mm nozzle for parts that need to be very strong.
